internal/time.go

```diff
@@ -33,20 +33,20 @@ func replaceTimes(v interface{}) interface{} {
 
 	m, isMap := v.(map[string]interface{})
 	if isMap {
-		copy := util.CopyMap(m)
+		copyMap := util.CopyMap(m)
 		for k, v := range m {
-			copy[k] = replaceTimes(v)
+			copyMap[k] = replaceTimes(v)
 		}
-		return copy
+		return copyMap
 	}
 
 	s, isSlice := v.([]interface{})
 	if isSlice {
-		copy := make([]interface{}, len(s))
+		copyInterface := make([]interface{}, len(s))
```

Comment:
Same here.

Reply:
Let's use `sliceCopy` instead? In hindsight 'interface' seems like incorrect wording.

Reply:
Yes, fine